app/views/admin/statistics_announcements/_search_results.html.erb

```diff
@@ -0,0 +1,59 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
+
+<% if paginator.blank? %>
+  <p class="govuk-body app-view-edition-search-results__no_documents">No <%= filter.options[:dates] %> statistics announcements found</p>
+<% else %>
+  <div class="govuk-table--with-actions">
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Title"
+        },
+        {
+          text: "Organisation"
+        },
+        {
+          text: "Release date"
+        },
+        {
+          text: "Announcement"
+        },
+        {
+          text: tag.span("View", class: "govuk-visually-hidden")
+        }
+      ],
+      rows:
+        paginator.map do |statistics_announcement|
+          [
+             {
+               text: tag.strong(statistics_announcement.title)
+             },
+             {
+               text: statistics_announcement.organisations.map(&:name).to_sentence
+             },
+             {
+               text: sanitize("#{statistics_announcement.release_date.to_date.strftime("%e&nbsp;%B&nbsp;%Y").html_safe}
+                              <br>
+                              #{statistics_announcement.release_date.strftime("%l:%M%P")}
+                              (#{statistics_announcement.confirmed? ? 'confirmed' : 'provisional'})"
+               ),
+             },
+             {
+               text:  if publication = statistics_announcement.publication
+                        sanitize("#{link_to(publication.title, [:admin, publication], class: "govuk-link")}
```

Comment:
not a big deal but we might want to use the tag method rather than raw html

```
  tag.p(link_to(publication.title, [:admin, publication], class: "govuk-link"), class: "govuk-!-margin-0") +
  "(#{publication.current_state})"
```

Reply:
Hmmm, p.tag puts a line break gap in between the publication title and the status, where as <br> just has the status on the next line. Inclined to leave as is for now.

Reply:
I think the `p` tag is OK here with the `govuk-!-margin-0` class and makes it more consistent with the other stuff on the page, e.g. in the "Release date" column. Not a big deal though.

Reply:
Cool, nice. Yes, that works well. Thank you.